lib/ClangImporter/ClangImporter.cpp

```diff
@@ -177,12 +177,12 @@ namespace {
       // so that (a) we use the same code as search paths for imported modules,
       // and (b) search paths are always added after -Xcc options.
       SearchPathOptions &searchPathOpts = Ctx.SearchPathOpts;
-      for (const auto &framepath : searchPathOpts.FrameworkSearchPaths) {
+      for (const auto &framepath : searchPathOpts.getFrameworkSearchPaths()) {
         Importer.addSearchPath(framepath.Path, /*isFramework*/true,
                                framepath.IsSystem);
       }
 
-      for (auto path : searchPathOpts.ImportSearchPaths) {
+      for (auto path : searchPathOpts.getImportSearchPaths()) {
```

Comment:
I know you only changed to the getter here, but it's always nice to be explicit with `const auto &`. There's a few others where the same applies.